internal/compiler/object_tree.rs

```diff
@@ -319,6 +319,7 @@ pub struct Timer {
     pub interval: NamedReference,
     pub triggered: NamedReference,
     pub running: NamedReference,
+    pub element: ElementRc,
```

Comment:
I believe we need to keep a reference to the original element only to be able to lookup its index later as it is part of the .restart() function.
I'm wondering if there is a better way. I could think of storing the index within the `Element` Or rewriting the call to RestartTimer to have the timer index. But that seems dangerous.
I'm worried that the timer could be wrong after some inlining steps or something like that though. But it might be fine.